pyproject.toml

```diff
@@ -37,6 +37,7 @@ python = "^3.7"
 
 coverage = ">=5,<7"
 packaging = ">=20.4"
+importlib_metadata = ">=3.6; python_version < '3.10'"
```

Comment:
```suggestion
importlib-metadata = { version = "*", python = "<3.8" }
```

Reply:
Hmm, using the package for python < 3.10 makes sense though: https://docs.python.org/3.8/library/importlib.metadata.html
According to the documentation it is "provisional" up until 3.10.

Reply:
Ok!

Reply:
This is a hack I use for other libs with `importlib`: https://github.com/wemake-services/dotenv-linter/blob/master/setup.cfg#L103-L105

Reply:
It's really bizarre though. I have the same version of mypy locally as is available in the venv. How can this yield differing results?!

Reply:
The test environment also installs that same version: https://github.com/wemake-services/coverage-conditional-plugin/actions/runs/3126867036/jobs/5072853018#step:5:96

Reply:
https://github.com/wemake-services/coverage-conditional-plugin/actions/runs/3127020506/jobs/5073182825#step:6:13 :shrug:

Reply:
Hm, so I can't reproduce this locally (I'm on python 3.10):

```
poetry run mypy coverage_conditional_plugin
Success: no issues found in 1 source file
```

I can't run the entire test, because mypy wants to check my system installation for unknown reasons: 
```
make test
poetry run mypy coverage_conditional_plugin tests/*.py
/usr/lib/python3.10/site-packages/numpy/__init__.pyi:636: error: Positional-only parameters are only supported in Python 3.8 and greater
Found 1 error in 1 file (errors prevented further checking)
make: *** [Makefile:5: lint] Error 2
```

Reply:
Ah, my bad, I messed up the mypy config it seems...